tests/runtime/engine/runner.py

```diff
@@ -93,6 +95,8 @@ def skip_reason(test, status):
             return "disabled by environment variable"
         elif status == Runner.SKIP_AOT_NOT_SUPPORTED:
             return "aot does not yet support this"
+        elif status == Runner.SKIP_IN_SKIPLIST:
+            return "disabled by entry in skiplist"
```

Comment:
super nit

Would also be cool to print the file name or path but maybe that's too verbose
```suggestion
            return "disabled by entry in skiplist file"
```